### DISCUSSION TITLE
Node.red dashboard link speichern

### DISCUSSION THREAD

**Discussion by oelidoc** [camueller/SmartApplianceEnabler#554] [Installation und Betrieb]
Hallo,
ich habe heute node-Red mit dem Dashboard installiert, um mal die Anlaufströme etwas genauer zu untersuchen. Läuft auch soweit alles.
Vielen Dank dafür!
Allerdings gelingt es mir nicht, die Dashboard URL in den Einstellungen des SAE 2.3.0 einzustellen. Egal, was ich eingebe, er springt immer wieder auf die ausgegraute Vorbelegung http://localhost:1880/ui zurück. Und unter dieser URL kommt keine Verbindung zum Dashboard zustande - warum auch immer.
Was mache ich falsch? Warum funktioniert http://localhost:1880/ui nicht?
Vielen Dank
oelidoc

**Comment by camueller** [comment]
Ich habe das gerade mal getestet und konnte als Dashboard-URL "http://www.google.de" und auch "http://www.google.de/ui" eingeben. Was genau versuchst Du einzugeben? Und was ist auf der Seite vorher schon alles eingegeben? Nach dem Eingeben musst Du natürlich noch "Save" drücken. Ist dieser Button vielleicht ausgegraut weegen ungültiger Eingaben?

**Comment by oelidoc** [comment]
Also ich gebe auf der Seite Einstellungen bei Node-Red jetzt http://www.google.de ein und speichere. Dann wechsele ich auf die Status Seite und klicke auf den Dashboard Link: es öffnet sich der Chrome Browser und als Adresse wird http://localhost:1880/ui angezeigt, was nicht funktioniert ("die Website ist nicht erreichbar"), obwohl Red-Node und das Dashboard auf dem selben Raspi wie SAE laufen. Wenn ich dann zurückgehe auf die Seite Einstellungen steht dort wieder unter Red-Node die Vorbelegung http://localhost:1880/ui in hellem Grau. Also http://www.google.de wird nicht verwendet und auch nicht gespeichert. Auf der Seite Einstellungen sind noch der MQTT Broker mit einer IP 168.178.168... Port 1833 und das notifyWithSignal.sh Script eingetragen. Beides funktioniert, nur der Port des Brokers war noch nie vollständig lesbar: die letzte Ziffer ist immer nahezu vollständig abgeschnitten. SAE ist rel. frisch installiert, nutzt als MQTT Broker einen MQTT2_Server unter FHEM und läuft unter Port 8090, da unter 8080 bereits Zigbee2MQTT läuft.
Ich hoffe ich hab mich verständlich ausgedrückt und es hilft bei der Fehlersuche.
Gruß
oelidoc

**Comment by camueller** [comment]
Es scheint so, als ob bei Dir das Speichern nicht funktioniert. Du kannst in der `Appliances.xml` die URL für Node-RED auch manuell eintragen mit einem Editor. Das musst dann so aussehen:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Appliances xmlns="http://github.com/camueller/SmartApplianceEnabler/v2.0">
    <Configuration param="NodeRED.Dashboard.Url" value="http://raspi2:1880/ui"/>
[...]
```

**Comment by oelidoc** [comment]
Das funktioniert!
Problem gelöst,
Vielen Dank für die prompte Hilfe und SAE überhaupt
Gruß
oelidoc
